src/convert.cpp

```diff
@@ -85,7 +84,7 @@ ddwaf_object* to_ddwaf_object(ddwaf_object *object, Napi::Env env,
   mlog("starting to convert an object");
   if (depth >= DDWAF_MAX_MAP_DEPTH) {
     mlog("Max depth reached");
-    return ddwaf_object_invalid(object);
+    return ddwaf_object_map(object);
```

Comment:
Worth adding a comment and even link to https://github.com/DataDog/libddwaf/issues/14

Reply:
I'll make an issue instead